src/test/java/com/sematext/solr/redis/TestRedisQParserPluginIT.java

```diff
@@ -7,23 +7,34 @@
 import org.apache.solr.common.params.SolrParams;
 import org.apache.solr.request.SolrQueryRequest;
 import org.junit.After;
+import org.junit.AfterClass;
 import org.junit.Before;
 import org.junit.BeforeClass;
 import org.junit.Test;
 import redis.clients.jedis.Jedis;
+import redis.embedded.RedisServer;
+
 import java.net.URLDecoder;
 import java.util.concurrent.TimeUnit;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 public class TestRedisQParserPluginIT extends SolrTestCaseJ4 {
   private static final Logger log = LoggerFactory.getLogger(TestRedisQParserPluginIT.class);
-
+  
   private Jedis jedis;
+  
+  private static RedisServer redisServer;
 
   @BeforeClass
   public static void beforeClass() throws Exception {
     initCore("solrconfig.xml", "schema.xml");
+
+    redisServer = RedisServer.builder()
+    		.port(6379)
```

Comment:
I think you used tabs and I am pretty sure in all places there are spaces.

Reply:
Oh, right. I didn't notice that. It should be fixed everywhere now.